sdk/identity/azure-identity/samples/azure-aad-auth-with-redis-py.md

```diff
@@ -36,7 +36,7 @@ Integrate the logic in your application code to fetch an Azure AD access token v
 import redis
 from azure.identity import DefaultAzureCredential
 
-scope = "https://*.cacheinfra.windows.net:10225/appid/.default"  # The scope will be changed for AAD Public Preview
+scope = "acca5fbb-b7e4-4009-81f1-37e38fd66d78/.default"  # The scope will be changed for AAD Public Preview
```

Comment:
The comment can be updated to "The current scope is for public preview and may change for GA release."